releasenotes/notes/process-agent-run-in-core-agent-linux-c19289949eb0d602.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Experimental: The process-agent checks(process, container, and process-discovery) can be run from the core agent in
```

Comment:
```suggestion
    Experimental: The process-agent checks(process, container, and process-discovery) can be run from the core Agent in
```

Reply:
@urseberry Should it be `Core Agent`?

Reply:
@wiyu Just chiming in because you pinged in #documentation, I think it should be `core Agent`.

Reply:
🙇